serverless.yml

```diff
@@ -138,3 +152,12 @@ custom:
   aliases:
     staging: staging-discretionarybusinessgrants.hackney.gov.uk
     production: discretionarybusinessgrants.hackney.gov.uk
+
+  staging:
```

Comment:
These should be under a label similar to aliases

Reply:
Got a quick minute to jump in a call? I'll show you why they're at this level.

Reply:
Ah do you mean under a subnet label?